compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -152,6 +152,140 @@ tests tools@Tools{damlc,validate,oldProjDar} = testGroup "Data Dependencies" $
               (if targetLfVer /= depLfVer then 2 else 0) -- different daml-stdlib/daml-prim
     | (depLfVer, targetLfVer) <- lfVersionTestPairs
     ] <>
+    [ testCaseSteps ("Cross DAML-LF version with stdlib orphan instances: " <> LF.renderVersion depLfVer <> " -> " <> LF.renderVersion targetLfVer)  $ \step -> withTempDir $ \tmpDir -> do
```

Comment:
it’s probably been too long since I looked at these tests but why can we not use `simpleImportTest` here?

Reply:
I think but didn't test that this has to be cross DAML-LF versions, since otherwise the stdlib _will_ be there. I'll test that now, though.

Reply:
I've tested this now and indeed the error doesn't show up if the data-dep is on the same DAML_LF version as the main package